
> simple-counter-test@1.0.0 deploy
> tsx deploy.ts


node:internal/modules/run_main:123
    triggerUncaughtException(
    ^
Error [TransformError]: Transform failed with 1 error:
/home/jeyem/Socious/simple-counter-test/deploy.ts:246:4: ERROR: Expected "finally" but found "else"
    at failureErrorWithLog (/home/jeyem/Socious/simple-counter-test/node_modules/esbuild/lib/main.js:1467:15)
    at /home/jeyem/Socious/simple-counter-test/node_modules/esbuild/lib/main.js:736:50
    at responseCallbacks.<computed> (/home/jeyem/Socious/simple-counter-test/node_modules/esbuild/lib/main.js:603:9)
    at handleIncomingPacket (/home/jeyem/Socious/simple-counter-test/node_modules/esbuild/lib/main.js:658:12)
    at Socket.readFromStdout (/home/jeyem/Socious/simple-counter-test/node_modules/esbuild/lib/main.js:581:7)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at Pipe.onStreamRead (node:internal/stream_base_commons:189:23)

Node.js v22.17.0
